### DISCUSSION TITLE
Meter signatures

### DISCUSSION THREAD

**Discussion by rettinghaus** [music-encoding/music-encoding#826] [General]
There are several tickets on meter signatures regarding rendition and grouping. 
As I plan to add some more to those, I think it would be good to have a platform where we can bring all those things together and have a look at the bigger picture.

* #504
* #697
* #818
* #827
* #970
* #987

Somewhat connected also:
* #700
* #322
* #848

Already addressed:
* #841
* #843

**Comment by ahankinson** [comment]
I'm not sure how #800 relates to meter signatures? Could you elaborate?

**Reply by rettinghaus** [reply]
Sorry, I meant #785, as Perry suggested something like `meterSigGrp` there.

**Comment by bwbohl** [comment]
2021-06-24 Odd Thursday: @rettinghaus thanks for moving this to the discussion and linking to potentially relevant issues. It seems that a thorough discussion and decision making process is good before taking any direct modifications of the schema. Synthesising all the above in on general concept and a PR might be good!

**Comment by annakijas1** [comment]
Here is another example of a meter signature that does not seem to have a good option for encoding and rendering accurately in Verovio. It is from Perkinson’s “Jettin’ Blue/s.”
<img width="125" alt="Screen Shot 2022-06-01 at 2 15 38 PM" src="https://user-images.githubusercontent.com/6841932/171917404-e907d61f-a387-48dd-98eb-9bf714b6125a.png">
